pandas/tests/extension/test_numpy.py

```diff
@@ -208,10 +208,9 @@ def test_series_constructor_scalar_with_index(self, data, dtype):
 
 
 class TestDtype(BaseNumPyTests, base.BaseDtypeTests):
-    @pytest.mark.skip(reason="Incorrect expected.")
-    # we unsurprisingly clash with a NumPy name.
+    @pytest.mark.skip(reason="PandasArray expectedly clashes with a NumPy name.")
```

Comment:
out of curiosiy which name?

Reply:
Object. This works for float so I changed to fail for the object dtype